edx_notifications/base_data.py

```diff
@@ -210,7 +210,7 @@ class DateTimeField(TypedField):
     Specialized subclass of TypedField(datetime) as a convienence
     """
 
-    _expected_types = [datetime]
+    _expected_types = [datetime, FakeDatetime]
```

Comment:
Hmmm. This is an unfortunate artifact of mocking out the system timer. Not sure if there's something better to do here....